nebula/ui/components/VPNServerCountry.qml

```diff
@@ -18,7 +18,10 @@ VPNClickableRow {
     property string _countryCode: code
     property var currentCityIndex
     property alias serverCountryName: countryName.text
-    property bool showConnectionScores: (focusScope.currentServer.whichHop === "singleHopServer")
+
+    // The city connection score can be used for every case except the multihop exit location,
+    // where we need to use the scoring between the entry and exit locations instead.
+    property bool useMultiHopScore: (focusScope.currentServer.whichHop === "multiHopExitServer")
```

Comment:
Can we have an enum?

Reply:
The `whichHop` string is used in a bunch of places, including our glean telemetry, so it's not a trivial refactor.